packages/core/src/database/postgres/service.ts

```diff
@@ -785,6 +792,10 @@ export class PostgresDatabaseService implements BaseDatabaseService {
           columnName,
           scalarToSqlType[column[" scalar"]],
           (col) => {
+            const sCol = column as ScalarColumn;
```

Comment:
Should `ReferenceColumn`s also support `.default`? I think it makes sense because they already support `.optional`

Reply:
i think you are right, that relationals could use a default value as well